Makefile

```diff
@@ -27,6 +27,7 @@ clean:
 publish: clean copy-examples
 	mkdir -p build
 	npm run publish 2> /dev/null
+	poetry run python scripts/validate_oas_examples.py
```

Comment:
Ideally this would run at the end of 'publish', so I chose to add this and remove the 'validate-examples' target. Makefile targets can only be dependent on other targets that execute **_before_**. 

We could consider removing this line and adding
validation-examples:
    poetry run python scripts/validate_oas_examples.py
but I guess we'd have to rely on the pipeline and developers to run this.

I considered other options, but they'll have secondary redundant effects.

We also should consider whether we want to always run this validation. If it breaks or fails, 'make publish' will fail.

Reply:
I'll let @nhsd-jack-wainwright review this as it seems he knows more about make files than me.

Reply:
I think this is fine, as part of publishing the OAS file we should be checking that the included examples are valid. It does mean that to run the validation locally you would also need to publish the OAS file, but publishing the OAS file is quite quick so I don't think that's too bad.